android/app/src/main/java/com/expensify/chat/CustomNotificationProvider.java

```diff
@@ -74,6 +80,32 @@ public CustomNotificationProvider(@NonNull Context context, @NonNull AirshipConf
         return builder;
     }
 
+    /**
+     * Creates a canvas to draw a circle and then draws the bitmap avatar within that circle
+     * to clip off the area of the bitmap outside the circular path and returns a circular
+     * bitmap.
+     *
+     * @param bitmap The bitmap image to modify.
+     */
+    public Bitmap getCroppedBitmap(Bitmap bitmap) {
+       Bitmap output = Bitmap.createBitmap(bitmap.getWidth(),
+            bitmap.getHeight(), Config.ARGB_8888);
+       Canvas canvas = new Canvas(output);
+
+       final int defaultBackgroundColor = 0xff424242;
```

Comment:
Is this color used only for masking? If this is shown on the notification then we need to make it theme-specific. 
e.g. Dark mode

Reply:
This color is the fill color for the circle that will be drawn in the background. I don't think that it will ever be shown as there will always be a bitmap drawn on top of it. But we could make it theme specific just in case.